### PR TITLE
Aerogear 3132

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation

Use fixed versions of platforms for showcase application to avoid constant refreshes that can affect stability.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
     "lint": "ionic-app-scripts lint",
     "ionic:build": "ionic-app-scripts build",
     "ionic:serve": "ionic-app-scripts serve",
-    "ionic:android": "ionic cordova run android --emulator --no-fetch",
```
Comment (ciaranRoche) [inline package.json]: Should we keep the --no-fetch flag to avoid losing sym links when using local sdk version

Comment (wtrocki) [reply package.json]: What version of cordova are you using? --no-fetch is no longer supported in cordova 8 which is what is being used by default.

**Comment by aliok** [comment]
Trying now.

**Comment by wtrocki** [comment]
@aliok I is still not finished. Js-sdk changes are required.

File: package.json (hunk 1 above)
Comment (ciaranRoche) [reply package.json]: Yeah I am using version 8, good catch, we might want to make an update to the docs, to make note of it for anyone developing the SDK, or are we only supporting cordova 8?

**Comment by aliok** [comment]
Ok, was stuck at some other thing anyway. @wtrocki let me know when it's ready for verification.

**Comment by wtrocki** [comment]
It should be ok now!

File: package.json [hunk 2]
```diff
@@ -10,8 +10,8 @@
...
-    "ionic:ios": "ionic cordova run ios --emulator --no-fetch",
```
Comment (StephenCoady) [inline package.json]: is there a reason we want to remove --emulator by default? seems pointless as most people will run it using an emulator for quick dev

Comment (StephenCoady) [reply package.json]: nevermind, @ciaranRoche has informed me it looks for an emulator if there is no device present. kudos!

**Pull request merged**